.theia/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "editor.autoSave": "on"
+}
```

Comment:
You shouldn't have this here

Reply:
Means I should comment it out ?? I didn't understand actually like what are you trying to say